src/agenteval/log.py

```diff
@@ -41,12 +41,19 @@ def compute_model_cost(model_usages: list[ModelUsageWithName]) -> float:
     """
     total_cost = 0.0
     for model_usage in model_usages:
+
+        prompt_tokens = model_usage.usage.input_tokens
+        # anthropic results split total tokens across input, output, cache write, and cache read values
+        if model_usage.usage.total_tokens != prompt_tokens + model_usage.usage.output_tokens:
+            prompt_tokens += (model_usage.usage.input_tokens_cache_write + model_usage.usage.input_tokens_cache_read)
```

Comment:
Could probably re-check against total_tokens after this increment to be extra safe